src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerGetAdminMediasByIdIntTest.java

```diff
@@ -271,6 +273,65 @@ void shouldDenyAccess(SecurityRoleEnum role) throws Exception {
                                     """, jsonString, JSONCompareMode.STRICT);
     }
 
+    @Test
+    void getMediaById_shouldReturnCaseWithCasIdAndActualHearing_whenMediaHasLinkedCase() throws Exception {
+        // Given
+        given.anAuthenticatedUserWithGlobalAccessAndRole(SecurityRoleEnum.SUPER_USER);
+        UserAccountEntity userAccountEntity = databaseStub.getUserAccountRepository().findAll().stream()
+            .findFirst()
+            .orElseThrow();
+
+        // Create hearing and mark it as actual
+        var hearingEntity = databaseStub.createHearing(COURTHOUSE_NAME, COURTROOM_NAME, CASE_NUMBER, HEARING_START_AT.toLocalDateTime());
+        hearingEntity.setHearingIsActual(true);
+        databaseStub.getHearingRepository().save(hearingEntity);
+
+        // Create media and link it to hearing
+        var mediaEntity = createAndSaveMediaEntity(hearingEntity, userAccountEntity, false);
+
+        // Create media linked case with court case (AC1)
+        databaseStub.createMediaLinkedCase(mediaEntity, hearingEntity.getCourtCase());
+
+        // When
+        mockMvc.perform(get(ENDPOINT.resolve(String.valueOf(mediaEntity.getId()))))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.cases", hasSize(1)))
+            .andExpect(jsonPath("$.cases[0].id").value(hearingEntity.getCourtCase().getId()))
+            .andExpect(jsonPath("$.cases[0].case_number").value(CASE_NUMBER))
+            .andExpect(jsonPath("$.cases[0].source").value("Legacy"))
+            .andExpect(jsonPath("$.cases[0].courthouse.id").value(hearingEntity.getCourtCase().getCourthouse().getId()))
+            .andExpect(jsonPath("$.cases[0].courthouse.display_name").value(COURTHOUSE_NAME));
+    }
+
+    @Test
+    void getMediaById_shouldReturnCaseWithoutCasId_whenMediaHasMigratedCase() throws Exception {
```

Comment:
Maybe update the test name? 'MigratedCase' won't mean anything to a future dev so maybe update to how this relates to modernised?

Reply:
now renamed